SR2025/2025-01-15-check-in.md

```diff
@@ -0,0 +1,27 @@
+---
+to: SR2025 teams
+subject: Student Robotics 2025 {TLA} January Check-in
+---
+
+Hi {PrimaryGivenName},
+
+Happy New Year!
+
+Just wanted to check in on how your team is doing?
+
+It would be great to hear what your team is currently working on and if there's
+anything you'd like a hand with.
+
+A couple of quick reminders for the next few weeks:
+
+* if you'd like to come to the [Cambridge Tech Day][cambridge-tech-day] on
+  Saturday please [let us know][tech-day-signup] this week
+* the second [Challenge Deadline][challenge-deadline] is also coming soon -- get
+  your submissions in by 6pm Saturday 25th to earn another 6 league points
+
+Thanks,\
+Peter
```

Comment:
Issue: In the past, we've intentionally avoided personally signing our names on emails from `teams@`, since really the emails are from the organisation.

Reply:
For mass-emails yes, however for replies I've noticed we do tend to. This email is intended to feel like a direct email (even though it's templated) in order to try to get more responses to the core question of how they're doing.